### PR TITLE
Sharing chat UI functionality

### PR DESCRIPTION
**Pull request by cristiandouce**

### CODE REVIEW

File: components/auth0/connected-accounts.tsx [hunk 1]
```diff
@@ -1,19 +1,12 @@
 "use client";
 
-import { useCallback, useEffect, useState } from "react";
 import { useRouter } from "next/navigation";
+import React, { useCallback, useEffect, useState } from "react";
```
Comment (cristiandouce) [inline components/auth0/connected-accounts.tsx:4]: the only changes on this file are

1. Adding `React` to fix lint error
2. Remove `<Toaster />` component now loaded in higher level at `app/layout.tsx` instead.

File: components/header.tsx [hunk 2]
```diff
@@ -1,59 +0,0 @@
-import Link from "next/link";
```
Comment (cristiandouce) [inline components/header.tsx:1]: this was actually moved to `components/chat/header.tsx` 🤔

File: components/icons.tsx [hunk 3]
```diff
@@ -625,37 +625,98 @@ function ChevronRightIcon() {
...
-      viewBox="0 0 32 32"
+      xmlns="http://www.w3.org/2000/svg"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
+      stroke="currentColor"
+      stroke-width="2"
+      stroke-linecap="round"
+      stroke-linejoin="round"
```
Comment (cristiandouce) [inline components/icons.tsx:528]: need fix 😞

File: components/icons.tsx [hunk 4]
```diff
@@ -625,37 +625,98 @@ function ChevronRightIcon() {
...
...1 3.27257 8.35757 3.11027 8.51194 3.00541ZM4.31796 7.14672C4.51322 6.95146 4.51322 6.63487 4.31796 6.43961C4.12269 6.24435 3.80611 6.24435 3.61085 6.43961L2.99213 7.05833L2.94922 7.10124C2.57957 7.47077 2.34303 7.70724 2.17788 7.95035C1.50265 8.94432 1.4678 10.238 2.11799 11.2281C2.27656 11.4695 2.50766 11.7005 2.8591 12.0518L2.90374 12.0965L2.94837 12.1411C3.29967 12.4925 3.53068 12.7237 3.77214 12.8822C4.76219 13.5324 6.05589 13.4976 7.04986 12.8223C7.29296 12.6572 7.52943 12.4206 7.89896 12.051L7.89897 12.051L7.94188 12.0081L8.5606 11.3894C8.75586 11.1941 8.75586 10.8775 8.5606 10.6823C8.36533 10.487 8.04875 10.487 7.85349 10.6823L7.23477 11.301C6.80777 11.728 6.6423 11.8903 6.48794 11.9951C5.81158 12.4546 4.95642 12.4636 4.32107 12.0464C4.17681 11.9516 4.02274 11.8012 3.61085 11.3894C3.19896 10.9775 3.0486 10.8234 2.95385 10.6791C2.53661 10.0438 2.54561 9.18863 3.00507 8.51227C3.10993 8.35791 3.27224 8.19244 3.69924 7.76544L4.31796 7.14672ZM9.62172 6.08558C9.81698 5.89032 9.81698 5.57373 9.62172 5.37847C9.42646 5.18321 9.10988 5.18321 8.91461 5.37847L5.37908 8.91401C5.18382 9.10927 5.18382 9.42585 5.37908 9.62111C5.57434 9.81637 5.89092 9.81637 6.08619 9.62111L9.62172 6.08558Z"
```
Comment (cristiandouce) [inline components/icons.tsx:562]: 🤔  IDK why this one changed that much TBH

**Review by jfromaniello — APPROVED** [review]

**Pull request merged**